src/lucky/serializer.cr

```diff
@@ -1,3 +1,5 @@
+require "uuid/json"
```

Comment:
Nice! I didn't know this was a thing. I wonder if this should be included at a higher level so it's accessible in other areas of Lucky 🤔

Reply:
YES! Thanks for tracking that down @hanneskaeufler. Definitely agree we should just do that for people

Maybe could be in another spot, but I think it's probably ok here and we can move it if needed.